wrappers/Julia/CoolProp.jl

```diff
@@ -1,190 +1,677 @@
-VERSION < v"0.4.0" && __precompile__()
```

Comment:
Why do you remove this?
It should allows to pre-compile at first use than use directly the compiled version on the next calls (until an update where it compiles again)
This makes quicker loading of the library once compiled.